tests/system/testdata/authz/default/events.txtar

```diff
@@ -0,0 +1,44 @@
+user zumi
+
+ak project create --name pz
+return code == 0
+capture_jq zpid .project_id
+
+ak connection create c1 --project $zpid --integration test
+return code == 0
+capture_jq cid .connection_id
+
+# will use default org
+ak event list 
+return code == 0
+
+# will use default org
+ak event list --integration test
+return code == 0
+
+ak event list --project pz
+return code == 0
+
+ak event list --connection $cid
+return code == 0
+
+ak event save --from-file event.json 
+return code == $RC_UNAUTHZ
+
+user gizmo
+
+ak event list --project $zpid
+return code == $RC_UNAUTHZ
```

Comment:
Shouldn't list for someone else's stuff also return NOT FOUND instead of UNAUTHZ?